Libraries/PeriphDrivers/Include/MAX32520/spi.h

```diff
@@ -643,6 +643,18 @@ void MXC_SPI_AbortAsync(mxc_spi_regs_t *spi);
  * @param   spi         Pointer to SPI registers (selects the SPI block used.)
  */
 void MXC_SPI_AsyncHandler(mxc_spi_regs_t *spi);
+
+/**
+ * @brief   Enable disable HW CS control feature.
+ *
+ * Depend on the application user might would like to drive slave select pin
+ * The SPI driver able to drive SS pin automatically this feature can be disable/enable
+ * by this function
+ *
+ * @param   spi             Pointer to SPI registers (selects the SPI block used.)
+ * @param   state           true or false
+ */
+void MXC_SPI_HWSSControl(mxc_spi_regs_t *spi, int state);
```

Comment:
```
/**
 * @brief   Enable/Disable HW CS control feature.
 *
 * Depending on the application, the user might need to manually drive the slave select pin.
 * The SPI driver automatically drives the SS pin and this function enables/disables this
 * feature.
 *
 * @param   spi             Pointer to SPI registers (selects the SPI block used.)
 * @param   state           true or false
 */
```
I revised the tone and grammar. Maybe something like this? ^^

Reply:
Fine, thanks.